src/lib.rs

```diff
@@ -28,10 +28,10 @@ pub static STANDARD: Config = Config {char_set: CharacterSet::Standard, pad: tru
 pub static URL_SAFE: Config = Config {char_set: CharacterSet::UrlSafe, pad: true};
 pub static URL_SAFE_NO_PAD: Config = Config {char_set: CharacterSet::UrlSafe, pad: false};
 
-
-#[derive(Debug, PartialEq, Eq)]
+#[derive(Debug)]
 pub enum Base64Error {
     Utf8(str::Utf8Error),
+    FromUtf8(string::FromUtf8Error),
```

Comment:
I actually agree that it isn't necessarily needed (and I don't use it myself) but I thought it was quite nice to have in terms of UX.
I'm not sure another `decode_` method is worth it, we can probably close down that PR